module/VuFind/src/VuFind/I18n/Translator/TranslatorAwareTrait.php

```diff
@@ -101,6 +101,16 @@ public function getTranslatorLocale($default = 'en')
      */
     public function translate($target, $tokens = [], $default = null)
     {
+        if ($this->getTranslatorLocale() == "keys") {
+            $keyValueToString = function ($key, $val) {
+                return "$key = $val";
+            };
+            $tokenDetails = empty($tokens)
+                ? ''
+                : '| [' . implode(', ', array_map($keyValueToString, array_keys($tokens), array_values($tokens))) . ']';
```

Comment:
I think this might look better if there were a space before the pipe -- i.e. `' | ['` instead of `'| ['`.

Reply:
You are right. I changed this.